custom_components/mail_and_packages/const.py

```diff
@@ -431,6 +431,20 @@
     },
     "dhl_parcel_nl_packages": {},
     "dhl_parcel_nl_tracking": {"pattern": ["[0-9A-Z]{12,24}"]},
+    # Bonshaw Distribution Network
+    "bonshaw_distribution_network_delivered": {
+        "email": ["parcel_tracking@bonshawdelivery.com"],
+        "subject": ["Parcel Delivered! Commande Livrée!"],
```

Comment:
They put both english and french in the subject line?

Reply:
They do indeed. Attached screenclip for reference. The company is based in New Brunswick which is legally the only official bi lingual province of Canada so they must show both.

![Parcel_Out_for_Delivery__En_attente_de_livraison__-_markwoodward86_gmail_com_-_Gmail](https://user-images.githubusercontent.com/47867906/218553450-c795fcd5-a46a-45c7-88d9-ea2b8f034e87.jpg)